Sources/XcbeautifyLib/Renderers/GitHubActionsRenderer.swift

```diff
@@ -0,0 +1,286 @@
+import Foundation
+
+struct GitHubActionsRenderer: OutputRendering {
+    private enum AnnotationType: String {
+        case notice
+        case warning
+        case error
+    }
+
+    let colored: Bool = false
```

Comment:
GitHub Actions annotations don't properly render colored output, so I decided to disallow it.